### ISSUE TITLE
Include a dayWrapper prop inside the month props.

### ISSUE THREAD

**Issue by saurishkar** [intljusticemission/react-big-calendar#782]
#### What's the current behaviour?
Currently, **month** view does not provide a wrapper for the individual days on the calendar. It will give more control to user, if a component for the same can be exposed.

Current props for the **month**:
 - header
 - dateHeader
 - event

#### What's the expected behaviour?
An extra prop required on the **month**:
 - header
 - dateHeader
 - event
 - **dayWrapper**

#### What's the benefit?
For starters, it can easily give control to users, which days to render based on holidays or weekends.
Can be many more instances.

**Comment by jquense** [comment]
Try dateCellWrapper ? That make work for you,

**Comment by saurishkar** [comment]
@jquense Yes **`dateCellWrapper`** indeed is an interesting prop.

I actually tried hiding my weekends using the above prop, but i see that the **`rbc-segment`** have dynamic widths. Since i cannot control those elements through any of the props, they will be calculating widths taking into account the weekend days and hence the positioning of my events will break.
Any suggestions ?

**Comment by jquense** [comment]
Yeah, that sort of thing is not feasible via something simple like visually hiding cells since event positioning and layout needs to know what days you see. The only real option is to make the month view handle arbitrary weeks sizes like time grid does. But that would take some work

**Comment by saurishkar** [comment]
@jquense ok, thanks for that insight. Is there any **workaround / hack** for now that can achieve this desired requirement ?
